spec/system/courses/report_spec.rb

```diff
@@ -350,7 +373,7 @@
     end
 
     # Archive target with verified submission for the student
-    let(:target_l3) do
+    let(:milestone_target_l3) do
```

Comment:
I guess this is not a milestone target, as we are not setting the fields

Reply:
My understanding was this would update the milestone_target_l3 created [here](https://github.com/pupilfirst/pupilfirst/pull/1414/files#diff-a4f1723a7478e1b9200a3d43dfe0811399e21ae5d2741969d99b926fc39cd2baR44). Is that not the case?

Reply:
Yes @shilendras I overlooked this without checking the complete view of the file.